11ty/definitions/dude.md

```diff
@@ -0,0 +1,20 @@
+---
+title: dude
+slug: dude
+defined: true
+reading:
+  - text: Etymonline entry
+    href: https://www.etymonline.com/word/dude
+---
+
+colloquially, a man; a discourse marker expressing masculine solidarity; an expression of astonishment when used alone
+
+## Issues
+
+Dude was originally used to refer to a man who paid a lot of attention to his appearance, but became more broadly used to describe any man as part of African American Vernacular English in the 1960s. It was subsequently appropriated by white men in the 1980s, particularly on the West Coast, and has since grown to be used broadly by white men and women as a noun as well as a discourse marker.
```

Comment:
```suggestion
"Dude" was originally used to refer to a man who paid a lot of attention to his appearance, but became more broadly used to describe any man as part of African American Vernacular English in the 1960s. It was subsequently appropriated by white men in the 1980s, particularly on the West Coast, and has since grown to be used broadly by white men and women as a noun as well as a discourse marker.
```
```suggestion
"Dude" was originally used to refer to a man who paid a lot of attention to his appearance, but became more broadly used to describe any man as part of African American Vernacular English in the 1960s. It was subsequently appropriated by white men in the 1980s, particularly on the West Coast, and has since grown to be used broadly by white men and women as a noun as well as a discourse marker.
```

Reply:
Whenever I use the word in question (if it is an "avoid" word, I put it in quotations. (Reminds me to open issue for house-style guide.)